Assets/Scripts/Controllers/PlayerController.cs

```diff
@@ -108,7 +124,24 @@ public void OnTriggerExit(Collider collision)
             }
         }
     }
-    
+
+
+    protected override void Die()
```

Comment:
Couple of things about this method:

- Why set the player turn to false on death?
- The code block 132-138 is an exact duplicate of what we have in the parent method. Instead of doing this call base.Die()

Reply:
It's not a duplicate. Player dying triggers the game over screen, but not the bandit dying. The override contains OverlayManager.Endgame()